arches/app/templates/views/components/cards/default.htm

```diff
@@ -428,13 +430,15 @@ <h3 class="panel-title help-panel-title" style="">
                                     params: {
                                         state: 'report',
                                         preview: self.preview,
-                                        card: card
+                                        card: card,
+                                        showHeaderLine: $index !== 0
```

Comment:
@whatisgalen I think that the condition here should look something like this (as per your previous commit) so that there is also an hr between the tile data itself and the first card (if the tile itself has data):
```js
self.card.model.get("widgets")().length > 0 || $index() !== 0 
```